src/components/StructuredContent/StructuredContent.stories.mdx

```diff
@@ -0,0 +1,377 @@
+        "eiusmod tempor incididunt ut labore et dolore magna aliqua." +
+        "Pellentesque massa placerat duis ultricies lacus sed turpis tincidunt. " +
+        "<a href='#'>Porttitor eget dolor</a> morbi non arcu risus quis varius." +
+        "Faucibus in ornare quam viverra orci sagittis.</p><p>Nisl vel pretium lectus" +
+        "quam id leo in. Etiam dignissim diam quis enim lobortis scelerisque fermentum." +
+        "Diam maecenas sed enim ut sem viverra. <b>Diam quam nulla porttitor massa" +
+        "id neque. Sed faucibus turpis in eu mi.</b></p><p>Ornare lectus sit amet est placerat" +
+        "in. Quis blandit turpis cursus in. Aliquam ut porttitor leo a diam sollicitudin" +
+        "tempor id eu. Pellentesque eu tincidunt tortor aliquam nulla facilisi cras" +
+        "fermentum. Porttitor leo a diam sollicitudin tempor id eu nisl nunc. Feugiat" +
+        "nisl pretium fusce id velit ut tortor. Porttitor leo a diam sollicitudin " +
+        "tempor id eu nisl nunc.</p><ul><li>one</li><li>two</li>" +
+        "<li>three</li><li>four</li></ul><p><i>Mauris nunc congue nisi vitae suscipit" +
+        "tellus mauris a diam. Purus gravida quis blandit turpis cursus in hac." +
+        "Morbi tempus iaculis urna id volutpat. Lectus nulla at volutpat diam ut" +
+        "venenatis. Donec ac odio tempor orci dapibus ultrices in iaculis. Dui vivamus" +
+        "arcu felis bibendum ut tristique. Cras semper auctor neque vitae tempus" +
+        "quam pellentesque. Placerat orci nulla pellentesque dignissim enim sit" +
+        "amet. Feugiat pretium nibh ipsum consequat. Placerat orci nulla pellentesque" +
+        "dignissim. Suspendisse faucibus interdum posuere lorem. Nullam non nisi" +
+        "est sit. Turpis egestas integer eget aliquet nibh praesent. Tortor at risus" +
+        "viverra adipiscing at. Eu augue ut lectus arcu bibendum at varius vel" +
+        "pharetra.</i></p>"
+      }
+    />
+  </Story>
+</Canvas>
+
+## With HTML Element Text Content
+
+The following content is passed as HTML DOM elements to the `textContent` prop.
+
+<Canvas>
+  <Story name="With HTML Element Text Content">
+    <StructuredContent
+      calloutContent="This is the callout text"
+      headingText="Heading text"
+      imageAlt="Image alt text"
+      imageAspectRatio={ImageRatios.Original}
+      imageCaption="Image caption"
+      imageCredit="Image credit"
+      imagePosition={StructuredContentImagePosition.Left}
+      imageSize={ImageSizes.Medium}
+      imageSrc="https://placeimg.com/400/300/animals"
+      textContent={
+        <>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua.
+            Pellentesque massa placerat duis ultricies lacus sed turpis
+            tincidunt.
+            <a href="#">Porttitor eget dolor</a> morbi non arcu risus quis varius.
```

Comment:
Bring this line up with the end of the previous sentence.  When it's broken like this, the space between the `.` and the link text is lost.